README.md

```diff
@@ -22,12 +22,38 @@ Your goal is to produce shapes by cutting, rotating, merging and painting parts
+- On win10, you should see a list. Click `New` then type in `C:\Program Files\FFmpeg\bin`
+- On win7, you should see a window that says `Edit User Variable.` In the text field that says "Variable value:" you want to type at the end `;C:\Program Files\FFmpeg\bin`. The semicolon is for seperating the file path from the other file paths in that field.
+7. FFmpeg should now be installed. To test if you did this correctly, run `cmd.exe` as administrator and type `ffmpeg -version` and it should spit at you several lines of code.
+8. Install `Node.js`, `Yarn`, and `GitHub CLI`. All three of these softwares use setup wizards, so installation should be easy. If you don't know what setting to put in, use the default. Also make sure you have `Java` and `Git`, but both are very commom afaik. Run `cmd.exe` if you closed it earlier and run `java -version` and `git --version` to test if you already have them installed or not.
+9. `cd C:\Program Files (x86)\GitHub CLI` then `gh` in `cmd.exe`. Login through that, selecting HTTP over SSH.
+10. `cd %USERPROFILE%` to return to your user folder.
+11. `gh repo clone tobspr/shapez.io` to make sure the game's code is on your system.
+12. `git remote add upstream https://github.com/tobspr/shapez.io.git` for resetting branches to the current and official build on GitHub.
+13. Set up branches for development. If you just want to build and not make anything, skip this step. On your fork, you should create a new branch. Each unique pull request should have its own branch. Reserve the `master` branch to being level with `tobspr/master`. Run `git reset --hard upstream/(username)/master && git push --force` to reset your master to be level with tob's. Make sure all of your current work is on the branch you created.
+
+**EXECUTION**
+1. Start `cmd.exe`
+2. If you aren't already on `C:\Users\[Your user]`, type `cd %USERPROFILE%`
+3. `cd shapez.io`
+4. See below for switching builds. If you wish to perform any of those commands, run them at this step.
+5. `yarn & cd gulp & yarn & yarn gulp`. This will take a while, but afterwards shapez.io should be running in a browser tab. Note that if both `yarn` commands respond with `Already up-to-date`, if the future you only have to run `cd gulp & yarn gulp`, but only do this if you know you are up to date.
+
+**SWITCHING BUILD**
+- If you want to run a specific pull request, run `gh pr checkout [pr number]`
+- If you want to run a specific fork, run `git remote add [arbitrary name] [HTTP url, click 'code' on the fork's page and copy the url here] & git fetch -a`. You now have the fork as a remote. This will allow you to select the branches of the fork (see below). You only have to do this once per fork, and you can check the remotes you currently have with `git remote`.
+- If you want to run a specific branch, run `git branch -a` and choose the branch you want to run. Run `git checkout [branch]`. This should put you into `detached HEAD`.
```

Comment:
`detached HEAD` is not always the case (such as branches from running pull requests)